cds-snc/c19-benefits-node
# Feature/province list

# Description
# Dropdown
Closes: https://trello.com/c/KcuVOorR/181-people-can-type-to-select-their-province-or-territory-from-a-list
Allow users to select their province or territory from a drop down list. It allows the users to type in the province or territory and will auto-complete to the appropriate value. It does not allow values not in the list.

![Screenshot from 2020-06-16 15-44-45](https://user-images.githubusercontent.com/16103529/84821730-8e6d7400-afe9-11ea-9ef3-7a2c14146b41.jpg)

## Test Instructions

1. Navigate to the Your province or territory page
2. Enter text in text field or click on arrow of dropdown
3. Validate that the correct result displays on the results page.

# Back Button

Modified implementation of the Back button. The back button now stores a history of pages viewed in the session so the user can navigate back to the last page viewed with javascript disabled. 

![Screenshot from 2020-06-16 16-06-49](https://user-images.githubusercontent.com/16103529/84822918-84e50b80-afeb-11ea-8fb1-20f43f59ad31.jpg)

## Test Instructions for back button changes

1. Navigate to the Your province or territory page
2. Select a province or territory and click Next
3. Validate that the back button appears at the top left of the current page
4. Click back button
5. Validate that it brings you back to the Your province or territory page
6. Validate that the back button does NOT appear on the Your province or territory page

## Help Requested
- Validate that the province drop down looks and behaves as expected.
